### ISSUE TITLE
V3 Status Fixes

### ISSUE THREAD

**Issue by marcelbertram** [porsche-design-system/porsche-design-system#2487]
Scope
---
As part of the V3 migration we still have some general and component specific improvement tasks.

Notes
---
- Grab some items from the list and document in subtasks and communicate in channel to avoid double work
- As part of the issue update the status overview file and remove completed topics
- https://designsystem.porsche.com/v3/component-v3-status
- use housekeeping branch

Acceptance Criteria
---
- [x] V3 Status fixes are closed and list updated

Out of scope
---
- Accordion Optimize icon (size) for compact mode #2309 
- Accordion hover state 
- [x] Radio focus is getting lost in some circumstances (#2206)
- [x] Naming of banner width should probably be aligned with what's going on in the carousel since there is similar behavior
- [x] Accordion Remove resize-observer fallback
- [x] Pagination Safari + Safari iOS: Prev button always looks disabled
- [x] Checkbox intermediate example only shows the first checkbox as intermediate.
- [x] Use px instead of rem apart from font-size and get rid of pxToRemWithUnit() where possible
- [x] modal ::before element has visible border (2px solid) #1971 
- [x] Add letter-spacing: normal; as fontLetterSpacing and reuse in typography styles (Name)
- [x] Link-Tile model signature Panamera is overlapping the padding of on small viewports 320px (https://github.com/porsche-design-system/porsche-design-system/issues/2487#issuecomment-1512890112)
- [x] Model signature: Change color in Safari when changing the theme programmatically
- [x] Toast External @Method() addMessage() must return a Promise
- [x] Internalt Rename utilities/projects/{styles/utilities}
- [x] Pagination: Get rid of counterReset (no JS for SSR) / Pagination shouldn't loose focus by tabindex="0" when prev/next buttons are getting aria-disabled="true" #578 

Subtasks
---
- [x] Update "Usage Docs missing" entries in "V3 Status" (Anastasiia) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2497)
- [x] Carousel -> Safari iOS: align-header="center" has no effect (Anastasiia) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2499)
- ~~[ ] Text|Heading|Display: Styling could be on :host instead of .root (Tatjana) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2502)~~
- [x] Text|Heading|Display: Reuse style across component
- [x] Checkbox/Radio: Safari while hovering on label the state does not change visually: Using :host(:hover) &(input:checked) would solve the issue but to hover is applied to the whole host element.? (Tatjana) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2508/files)
- [x] Checkbox in Safari Keyboard check/uncheck is not rendered immediately only after moving away or pressing a key (Tatjana) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2508/files)
- [x] Add vrt test for carousel, modal, banner, toast in 3000px (Henri) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2507)
- [x] Modal: close x button is using hover style of button primary dark theme to have a visible icon (https://github.com/porsche-design-system/porsche-design-system/issues/2487#issuecomment-1512893414) (Ole) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2510)
- [x] Button/Link Pure active&hover left padding when align-label is set to left: https://stackblitz.com/edit/6s9rfh?file=index.html (Ole) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2511)
- [x] Align how to accessible hide label/test in components with a utility for text-indent: -999999px; and apply to all components to get rid of other solutions (https://webaim.org/techniques/css/invisiblecontent/) (Henri) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2516)
- [x] ~~Playwright VRT doesn't render frostedGlass background effect. Maybe it's an WebKit/Linux thing? (Anastasiia)~~
- [x] Tabs bar focus of content is still in old focus style, see comment: https://github.com/porsche-design-system/porsche-design-system/issues/2487#issuecomment-1516460401 (Ole) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2521)
-  ~~Tabs(&Bar) size=small&medium spacing and scrolling-button alignment should always be vertically centered within the focus area and with the labels (currently too low), see comment: https://github.com/porsche-design-system/porsche-design-system/issues/2487#issuecomment-1516467552~~ won't fix, see comment: https://github.com/porsche-design-system/porsche-design-system/issues/2487#issuecomment-1521769459 (Ole)
- [x] Tabs(&Bar) active underline should be 2px instead of 1.5px to support non-retina (Ole) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2521)
- [x] Banner: in Safari + Safari IOS Storefront playground example does not show banner when scrolling because it is sticky (Tatjana) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2518/files)
- [x] Select: Tiny border-radius is visible when select overlay is open, see screen in comments: https://github.com/porsche-design-system/porsche-design-system/issues/2487#issuecomment-1516452478 (Henri) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2524)
- Textarea resize handle is not available in Safari (16.4.) (Ole)
  **--> Can't be fixed! No solution is available. The issue occurs due to the padding values >= 10px of the textarea. But it's not totally broken, because it works if the amount of text provokes displaying a scrollbar. If so, you can grab the resize handle. And this is probably the case when you want to resize a textarea...**
- [x] Icon deprecation warnings for deprecated lazy prop added (Ole) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2521)
- [x] Variant 'x-small' for icon in Safari isn't in sync with line-height of text, because of rounding errors caused by browser. Maybe it's fixable by experimental feature https://developer.mozilla.org/en-US/docs/Web/CSS/round, alternative use condition with fixed value (Ole) [PR](https://github.com/porsche-design-system/porsche-design-system/pull/2521)

**Comment by marcelbertram** [comment]
<img width="320" alt="image" src="https://user-images.githubusercontent.com/42934679/232759341-a51ba47b-f87e-475f-88ad-672502bc25ea.png">

**Comment by marcelbertram** [comment]
<img width="193" alt="image" src="https://user-images.githubusercontent.com/42934679/232759894-4aab4bea-be0a-4725-b598-d02ba8a5f53d.png">

**Comment by marcelbertram** [comment]
![Image](https://user-images.githubusercontent.com/42934679/233401106-cfdd441f-7f7e-4976-901b-a41ea27352fc.png)

**Comment by o-hook** [comment]
Due it looks a bit inconsistent, the alignement is correct. The white space at the bottom is reserved for the indicator bar. So we need to have a compromise here.
